src/parser/jobs/mnk/modules/Fists.js

```diff
@@ -168,7 +220,7 @@ export default class Fists extends Module {
 	}
 
 	output() {
-		const uptimeKeys = Object.keys(this._fistUptime).map(Number)
+		const uptimeKeys = Object.keys(this._fistUptime).map(Number).filter(fist => fist > 0)
```

Comment:
I'm not sure this is doing what you want it to - won't it unconditionally drop `STANCELESS` from the chart? You're filtering the keys, not the values. Assuming the goal is to remove 0s entries, I think the simpler solution is to just drop the initial `[STANCELESS]: 0` when you create the `_fistUptime` object and instantiate it as an empty blob instead.

Reply:
I think my intent here was to do `this._fistUptime[fist] > 0` but I honestly can't remember, this was sitting on my HDD for a few days before I committed it. There was a reason we initialised stanceless as zero but can't quite recall it, I think it was because pre-normaliser, the initial value would get confused if the user was actually stanceless. Do you think just making it an empty blob would work?